src/banking_stage.rs

```diff
@@ -109,12 +114,22 @@ impl BankingStage {
 
             debug!("process_transactions");
             let results = bank.process_transactions(transactions);
-            let transactions = results.into_iter().filter_map(|x| x.ok()).collect();
-            signal_sender.send(Signal::Transactions(transactions))?;
+            let mut transactions: Vec<Transaction> =
+                results.into_iter().filter_map(|x| x.ok()).collect();
+            tx.append(&mut transactions);
+            if tx.len() >= 512 {
```

Comment:
What's going on here?

Reply:
I'll add documentation. Transactions are coalesced into chunks of max 512 before it's sent to the next stage.